app/css/general.css

```diff
@@ -110,6 +114,14 @@ html,body {
 .button.current .icon{
 	-webkit-filter: invert(100%);
 }
+@keyframes fadeout {
+	from { opacity: 1; }
+	to   { opacity: 0; }
+}
+@keyframes fadein {
+	from { opacity: 0; }
+	to   { opacity: default; }
+}
```

Comment:
why do you `fadein` to `default` opacity, but `fadeout` from `1`? Does this mean all fadeout events will cause the element to jump to 100% opacity before fading?